CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to rpgp will be documented in this file.
 
+## 0.15.0 - Unreleased
+
+### 🚜 Refactor
+
+- Move x448 work behind an `unstable-curve448` feature, off by default - ([#426](https://github.com/rpgp/rpgp/pull/426))
```

Comment:
pleae remove this, changelog is auto generated when running the release process, based on the commit message

Reply:
I removed this section, though i don't know how to structure my commits so that they trigger the right kind of changelog update in the release process.  if you handle that, i'll look at your commits and try to emulate them in the future.